apps/openmw/mwrender/camera.cpp

```diff
@@ -42,18 +42,24 @@ class UpdateRenderCameraCallback : public osg::NodeCallback
 namespace MWRender
 {
 
+    const float Camera::sResetThirdPersonCameraDistance = 192.f;
+    const float Camera::sResetThirdPersonOverShoulderCameraDistance = 92.f;
+
     Camera::Camera (osg::Camera* camera)
     : mHeightScale(1.f),
       mCamera(camera),
       mAnimation(NULL),
-      mFirstPersonView(true),
+      mCameraView(CameraView_FirstPerson),
+      mActiveThirdPersonView(CameraView_ThirdPersonCenter),
       mPreviewMode(false),
       mFreeLook(true),
       mNearest(30.f),
       mFurthest(800.f),
       mIsNearest(false),
       mHeight(124.f),
-      mMaxCameraDistance(192.f),
+      mMaxCameraDistance(sResetThirdPersonCameraDistance),
+      mCameraXOffsetFromCenter(0.0f),
```

Comment:
You should put mCameraXOffsetFromCenter and mCameraZOffsetFromCenter initialization after mCameraDistance one, otherwise you will get "reorder" warning in GCC.